Makefile

```diff
@@ -200,13 +208,22 @@ isort:
 .PHONY: teardown
 teardown:
 	@echo "Tearing down the dev environment..."
-	@docker-compose down --remove-orphans
+	@docker-compose \
+	-f docker-compose.yml \
+	-f docker/docker-compose.integration-mariadb.yml \
+	-f docker/docker-compose.integration-mongodb.yml \
+	-f docker/docker-compose.integration-mssql.yml \
+	-f docker/docker-compose.integration-mysql.yml \
+	-f docker/docker-compose.integration-postgres.yml \
+	down \
+	--volumes \
```

Comment:
@sanders41 `init_db` is included as part of `start_webserver` so there shouldn't be an issue here, was able to run it locally and it looked fine after a `make teardown`

this also made it _much_ easier for me to test the integrations since it means everything gets rebuilt/reloaded from scratch